crates/sui-json-rpc/src/api.rs

```diff
@@ -84,18 +87,18 @@ pub trait RpcReadApi {
     async fn get_transactions_in_range(
         &self,
         /// the matching transactions' sequence number will be greater than or equals to the starting sequence number
-        start: GatewayTxSeqNumber,
+        start: TxSequenceNumber,
         /// the matching transactions' sequence number will be less than the ending sequence number
-        end: GatewayTxSeqNumber,
-    ) -> RpcResult<Vec<(GatewayTxSeqNumber, TransactionDigest)>>;
+        end: TxSequenceNumber,
+    ) -> RpcResult<Vec<TransactionDigest>>;
 
     /// Return list of recent transaction digest.
     #[method(name = "getRecentTransactions")]
     async fn get_recent_transactions(
         &self,
         /// maximum size of the result
         count: u64,
-    ) -> RpcResult<Vec<(GatewayTxSeqNumber, TransactionDigest)>>;
+    ) -> RpcResult<Vec<TransactionDigest>>;
```

Comment:
Should we simply remove this method since we have a breaking change here which changes the response type and this can be replaced by get_transactions(TransactionQuery::All, null, max_limit, descending)>

Reply:
removed